docker/install/ubuntu_install_python_package.sh

```diff
@@ -33,7 +33,7 @@ pip3 install --upgrade \
     Pillow==9.1.0 \
     psutil \
     pytest \
-    tlcpack-sphinx-addon==0.2.1 \
+    git+https://github.com/tlc-pack/tlcpack-sphinx-addon.git \
```

Comment:
just a note as we get to pinning py deps with poetry we'll still need to do the update workflow. so we could pin this here anyways.